### DISCUSSION TITLE
Digital/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#4] [Announcements]
# Digital/

https://soc.ustc.edu.cn/Digital/

**Comment by wintermelon008** [comment]
大家好！除了上面介绍的内容，本学期的实验文档我们还额外添加了**评论系统**。大家可以在各个界面下方找到类似的评论栏，登录自己的 GitHub 账号即可发表相应的评论。

### 怎么使用？

评论系统的输入采用 Markdown 格式。如果你之前没有用过 Markdown，可以简单地将其当做普通文本（txt）格式，直接输入文字并点击评论即可。如果你对 Markdown 语法有所了解，可以使用**加粗**、*斜体*、句内的 `code block` 等特殊格式，以及相应的标题结构。

~~请注意：评论发布后无法直接删除或修改。如果你担心自己的内容会给大家带来错误引导，请联系助教帮助你修改或删除。~~ 感谢 [iBug](https://github.com/iBug) 学长的提醒，实际上大家是可以删除或修改自己的评论的，具体操作可以参考评论区学长的回复。

除了留下自己的疑问，大家也可以解答其他同学的疑问。这是一个相互交流、相互合作的平台。我们鼓励**合理范围内**的讨论与思考~

### 其他交流方式

如果你没有或者无法登录 GitHub 账号也没关系。除了文档下方的评论系统，在实验 QQ 群、各小组 QQ 群中大家也可以提出自己的问题，我们将统一进行解答。

**欢迎大家在评论系统里畅所欲言！**

**Reply by melon008** [reply]
你也可以在对应的评论下方直接发表评论~

**Comment by wenjie8403** [comment]
赞

**Comment by iBug** [comment]
> 请注意：评论发布后无法直接删除或修改。如果你担心自己的内容会给大家带来错误引导，请联系助教帮助你修改或删除。

实际上，点击评论区上方加粗的 `N 条评论` 字样（如下图所示）即可打开位于 GitHub 上的 Discussion 讨论区。若你已经登录了 GitHub\.com，那么你可以直接在 GitHub 的网页上编辑或删除你的评论，无需联系助教或其他任何人。

> ![image](https://github.com/wintermelon008/USTC-DigitalLab-Discussion/assets/7273074/4a7fdd8e-0fe5-4a52-8371-f8f28eb12c26)

同理，你也无需通过课程主页上的 Giscus 控件登录评论，而可以直接在 GitHub\.com 的讨论区中发布、修改或删除回复，对应的内容也会自动反映到课程主页上。

例如，本条回复就由原作者自行编辑过。

**Reply by wintermelon008** [reply]
感谢学长指出这一问题！文档中已经进行了对应的修改。

**Comment by Wayne-Zeng357** [comment]
好评

**Comment by wintermelon008** [comment]
再次测试
